src/client_lib/pegasus_scanner_impl.cpp

```diff
@@ -61,8 +61,12 @@ int pegasus_client_impl::pegasus_scanner_impl::next(std::string &hashkey,
 {
     ::dsn::utils::notify_event op_completed;
     int ret = -1;
-    auto callback = [&](
-        int err, std::string &&hash, std::string &&sort, std::string &&str, internal_info &&ii) {
+    auto callback = [&](int err,
+                        std::string &&hash,
+                        std::string &&sort,
+                        std::string &&str,
+                        internal_info &&ii,
+                        uint32_t expire_ts_seconds) {
```

Comment:
I think it's a bug for not copying ttl in `copy_data`. Maybe we should copy it in default